common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -417,11 +416,9 @@
 
             DiscussionThreadListView.prototype.goHome = function() {
                 var url, $templateContent;
-                this.template = _.template($('#discussion-home-template').html());
-                $templateContent = $(this.template());
+                $templateContent = $(this.homeTemplate().toString());
                 $('.forum-content').empty().append($templateContent);
-                $('.forum-nav-thread-list a').removeClass('is-active').find('.sr')
-                    .remove();
+                $('.forum-nav-thread-list a').removeClass('is-active').find('.sr').remove();
```

Comment:
Thanks for spotting that. I didn't intend to change that, but did so while cleaning up a merge conflict. I'll revert it.